src/couch/test/couch_bt_engine_partition_downgrade_tests.erl

```diff
@@ -40,7 +40,7 @@ teardown({Ctx, Paths}) ->
     end, Paths).
 
 
-downgrade_test_() ->
+downgrade_test() ->
```

Comment:
Are we sure that this is right? eunit deferred tests are weird so i can never tell off hand if the trailing underscore is necessary.

Reply:
my intention is not to let this `downgrade_test_/0` to run because we remove the downgrade logic from PQ to non-PQ test.

Reply:
Aha. Then I'd just delete the test.